tests/integration_test.py

```diff
@@ -1073,6 +1082,64 @@ def runTest(self):
         self.assertTrue(sock.fileno() > -1)
 
 
+class TestRunContainerReadingSocket(BaseTestCase):
```

Comment:
It's a pep8 style that we avoid abbreviated variable names for the same reason we avoid single letter variables in for loops. Unless there's a very specific reason/context to not have it, this is not one of those times.

I appreciate the `next` conflicts with a reserved keyword, expanding it out to include more information is better than dropping a vowel. Thanks.